pkgs/tools/admin/nomachine-client/default.nix

```diff
@@ -17,26 +17,29 @@ in
             "https://download.nomachine.com/download/${versionMajor}/Linux/nomachine_${version}_${versionBuild_x86_64}_x86_64.tar.gz"
             "https://web.archive.org/web/https://download.nomachine.com/download/${versionMajor}/Linux/nomachine_${version}_${versionBuild_x86_64}_x86_64.tar.gz"
```

Comment:
@talyz do you manually submit these to archive.org? I tried it out and failed to archive with a 503 error. I think I got the right URL...

https://web.archive.org/save/https://download.nomachine.com/download/7.0/Linux/nomachine_7.0.211_4_x86_64.tar.gz

Reply:
No, they archive it automatically, but not right away. It seems to be available now, but they've changed the way you could specify the URL - previously, `https://web.archive.org/web/<URL>` got you the latest snapshot they had through a redirect, but now it looks like they use a javascript redirect instead, so I guess we need to specify the full URL with date :/

Reply:
Actually, I don't think the `archive.org` URL helps us anyway, since the `nomachine.com` doesn't issue a 404, but sends back a page, which gives a hash mismatch. Nix never tries the second URL, so we can get rid of it.

Reply:
Removed.